packages/attestation-service/src/server.ts

```diff
@@ -0,0 +1,28 @@
+import * as dotenv from 'dotenv'
```

Comment:
Nit: A more common name for this would be index.ts assuming it's the service's entry point

Reply:
Changed it. I personally dislike the generic `index` name, but don't feel strongly